test/test-basics.spec.js

```diff
@@ -181,4 +183,23 @@ describe('dag-cbor', () => {
     const encoded = bytes.fromHex('a3636261720363666f6f0163666f6f02')
     assert.throws(() => decode(encoded), /CBOR decode error: found repeat map key "foo"/)
   })
+
+  test('determine encoded length of obj', () => {
```

Comment:
These are both really nice examples of using this, I think they would go well on the README, would you mind adding something there that demonstrates the ability to do `encodedLength` and `decodeFirst`?